src/helics/helics_enums.h

```diff
@@ -204,6 +204,28 @@ typedef enum {
     helics_property_int_console_log_level = 274
 } helics_properties;
 
+/** enumeration of the multi_input operations*/
+typedef enum {
+    /** Time and priority order the inputs from the core library*/
+    helics_multi_input_no_op = 0,
+    /** all inputs are assumed to be boolean and all must be true to return true*/
+    helics_multi_input_and_operation = 1,
+    /** all inputs are assumed to be boolean and at least one must be true to return true*/
+    helics_multi_input_or_operation = 2,
+    /** sum all the inputs*/
+    helics_multi_input_sum_operation = 3,
+    /** sum all the inputs*/
```

Comment:
I thought input diff was doing something slightly different (subtracting all subsequent values from the 1st one?).

Reply:
yes.  Though if the output type is specified as a vector it will take a diff operation [I(N+1)-I(N),...]
I will fix the comment